### ISSUE TITLE
Goal Achieved Notification

### ISSUE THREAD

**Issue by cgmb** [jvamosi/Diversibee#45]
When the player makes enough profit, we could congratulate them and prompt them (though not force them) to go on to the next level. Preferably with a pretty graphic.

**Comment by jvamosi** [comment]
Good idea. Perhaps it could be a "happy bee" logo that says "Yay! You did it!" or some such. 

On a related note, my 7yr old daughter drew this concept mockup for y'all. She thought that the user might need prompting to plant trees initially (i.e., she needed a prompt...) and that an unhappy bee should show up after a given amount of nothing but blueberries had been planted to indicate to the user that they weren't making the bees happy without habitat. 
![bee](https://cloud.githubusercontent.com/assets/7888683/8111375/b1bdf06c-101e-11e5-9d3a-039a52f6f758.JPG)

**Comment by jvamosi** [comment]
The logo with the border looks bee-u-ti-ful, by the way.

**Comment by cgmb** [comment]
I would be nice to include a special notification if the user finds a better configuration than any we know of. For Levels 1 and 2, I'm pretty sure we've found the optimal patterns, but for Level 3 I'm not so sure.

**Comment by cgmb** [comment]
Meeting the goal would also be a good time to re-enforce the lesson that was to be taught.
